addons/account/tests/test_account_invoice_report.py

```diff
@@ -114,5 +115,5 @@ def test_invoice_report_multiple_types(self):
             [6,              6,              1],
             [-20,            -20,           -1],
             [-20,            -20,           -1],
-            [-600,           -600,          -1],
+            [600,           -600,          -1],
```

Comment:
missing a space (even if the alignment is weird because it should be on the decimal point, but don't change that)